tests/Microsoft.DotNet.XHarness.Tests.Runners.Tests/Microsoft.DotNet.XHarness.Tests.Runners.Tests.csproj

```diff
@@ -0,0 +1,73 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFramework>netcoreapp3.1</TargetFramework>
+
+  <IsPackable>false</IsPackable>
+</PropertyGroup>
+
+<ItemGroup>
+  <PackageReference Include="Moq" Version="4.13.1" />
+  <PackageReference Include="xunit.runner.utility" Version="2.4.1" />
+</ItemGroup>
+
+<ItemGroup>
+  <Compile Include="..\..\src\Microsoft.DotNet.XHarness.Tests.Runners\Core\Extensions.cs">
```

Comment:
Why not reference the whole project?

Reply:
I'm testing internals :/

Reply:
It can be done like this: https://github.com/dotnet/xharness/pull/134

But maybe I don't know all the caveats of this and we don't want that.